src/index.js

```diff
@@ -0,0 +1,289 @@
+
+import './index.scss'
+import IMask from 'imask';
+import { swiper } from './modules/swiper';
+import { openModal } from './modules/openModal';
+import { map } from './modules/map'
+import { handleClick,burgerChange } from './modules/scroll';
+
+const totalSum = document.querySelector('.tickets__inf__calc__amount__total')
+const radioBtns = document.querySelectorAll('input[type="radio"]')
+const totalModalSum = document.querySelector('.totalSumGeneral')
+
+const ticketsModalBasik = document.querySelector('.quantinyOfBasic')
+const ticketsModalSenior = document.querySelector('.quantinyOfSenior')
+
+const totalSumOfBasikModal = document.querySelector('.totalSumOfBasic')
+const totalSumOfSeniorModal = document.querySelector('.totalSumOfSenior')
+
+const basicPriceFirst = document.querySelector('.basicPriceFirst')
+const seniorPriceFirst = document.querySelector('.seniorPriceFirst')
+const basicPriceSecond = document.querySelector('.SumOfBasic')
+const seniorPriceSecond = document.querySelector('.SumOfSenior')
+
+const createStyleFlex = document.querySelector('.inputItems__entryTicket__value2')
+const blockRadioBtns = document.querySelector('.tickets__inf__calc__items')
+
+let sum = 30;
+function radioHelper(item,priceBasik, priceSenior ){
+  sum = (parseInt(countInput.value) * priceBasik) + (parseInt(countInput2.value) * priceSenior)
+  totalSum.innerHTML = `Total € ${sum}`
+  totalModalSum.innerHTML = sum;
+  selectTicket.value = item.value
+  valueTicketsSpan.innerHTML = item.innerHTML
+  valueTicketsSpan.innerHTML = item.innerHTML
+  totalSumOfBasikModal.innerHTML =  priceBasik * parseInt(countInput.value)
+  totalSumOfSeniorModal.innerHTML = priceSenior * parseInt(countInput2.value)
+
+  basicPriceFirst.innerHTML = priceBasik
+  seniorPriceFirst.innerHTML = priceSenior
+
+  basicPriceSecond.innerHTML = priceBasik
+  seniorPriceSecond.innerHTML = priceSenior
+
+  createStyleFlex.classList.remove('testGap')
+}
+function testRadio (arg) {
+  sum = 0
+  if((arg === 'first') || (arg === 'firstT')){
+    radioHelper(options[0],priceFirstTicketBasic,priceFirstTicketSenior)
+    return
+  } else if((arg === 'second') || (arg === 'secondT')){
```

Comment:
Ты добавил return нету смысла делать else. Когда код дойдет до return, то он выйдет из функции. Это тебе позволяет избравиться от if else if else if else. 

if else if else if else - считается антипаттерном и так делать неприветствуется

```suggestion
    return
  }
  
  if((arg === 'second') || (arg === 'secondT')){
```

Reply:
Избавься от if else if else if else везде в своем коде